src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -1454,4 +1454,10 @@
   <data name="MarkAttributesWithAttributeUsageCodeFix" xml:space="preserve">
     <value>Apply 'AttributeUsageAttribute'</value>
   </data>
-</root>
+  <data name="AddMissingInterpolationTokenMessage" xml:space="preserve">
+    <value>The string may have a missing interpolation token</value>
```

Comment:
What about something like `This string literal uses composite formatting but is not an interpolated string. Did you miss the '$' prefix?`

Reply:
@Evangelink This will trigger RS1032 because it's multi-sentence and doesn't end with a period. I believe this is a false-positive. Should we update RS1032 to handle both question marks and periods exactly the same? or just suppress the diagnostic?